### PR TITLE
#4674 Addition of DigiTrust vendor check for gdpr when cmp is present.

### PR DESCRIPTION
**Pull request by goosemanjack**
## Type of change
- [X ] Bugfix

## Description of change
This adds in the CMP vendor check for DigiTrust to the DigiTrust ID system module. When cmp is present and GDPR applies, this performs a check for consent specifically against the DigiTrust vendor ID to match behavior in the standalone DigiTrust ID system.

## Other information
Issue #4674 
https://github.com/prebid/Prebid.js/issues/4674

### CODE REVIEW

File: modules/digiTrustIdSystem.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ var noop = function () {
 
 const MAX_RETRIES = 2;
 const DT_ID_SVC = 'https://prebid.digitru.st/id/v1';
+const DT_VENDOR_ID = 65; // cmp gvlVendorId
```
Comment (samtingleff) [inline modules/digiTrustIdSystem.js]: this should be 64

File: test/spec/modules/digitrustIdSystem_spec.js [hunk 2]
```diff
@@ -5,9 +5,39 @@ import {
 
 let assert = require('chai').assert;
 let expect = require('chai').expect;
-
+const DIGI_VENDOR_ID = 65;
```
Comment (samtingleff) [inline test/spec/modules/digitrustIdSystem_spec.js]: also should be 64

**Comment by goosemanjack** [comment]
Touching issue. Please review.

**Comment by benjaminclot** [comment]
Giving consent by default (timeout or else) is not GDPR compliant.

**Comment by goosemanjack** [comment]
@benjaminclot Thanks for your feedback and bringing this up. I will have our team review the behavior and adjust as necessary for compliance as this is a secondary check on top of the Prebid GDPR check.

**Comment by goosemanjack** [comment]
Opened issue #4904

**Pull request merged**